src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/NetworkedComponents/TextMeshPro/TextMeshProBroadcasterBase.cs

```diff
@@ -59,6 +59,8 @@ protected override TextMeshProBroadcasterChangeType CalculateDeltaChanges()
 
         protected override void SendCompleteChanges(IEnumerable<SocketEndpoint> endpoints)
         {
+            previousText = this.textMesh.text;
+            previousProperties = new TextMeshProperties(textMesh);

```

Comment:
There was some weird behavior here where font details were sent prior to populating on this first call to SendCompleteChanges